hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/freon/HadoopFsGenerator.java

```diff
@@ -89,7 +90,11 @@ public Void call() throws Exception {
     }
 
     contentGenerator =
-        new ContentGenerator(fileSize, bufferSize, copyBufferSize);
+        new Builder()
```

Comment:
Nit: I think it's more readable if builder is qualified where it is being used (and also more consistent with usage in other generator classes).

```suggestion
        new ContentGenerator.Builder()
```